### ISSUE TITLE
Splitting the dataset into the Training set and Test set

### ISSUE THREAD

**Issue by ajallal88** [ajallal88/Machine-Learning#10]
Write a code to Splitting the dataset into the Training set and Test set
